### PR TITLE
Add IPv6 support by default

### PR DESCRIPTION
**Pull request by Saiv46**

### CODE REVIEW

**Review by rom1504 — CHANGES_REQUESTED** [review]
If the purpose of this change is to set host to undefined by default, I'd better you put host = undefined explicitly even if it means the same

**Comment by rom1504** [comment]
And can you point to the doc saying that undefined means "listen to all
ipv4 and ipv6 adresses" ? That seems unlikely to me

**Comment by rom1504** [comment]
Still better to do host = undefined to make it explicit it's optional

**Comment by rom1504** [comment]
https://github.com/PrismarineJS/node-minecraft-protocol/blob/8e4978f3793504435d1b8d7235173f2075ce94db/src/server.js#L53
host option goes there fyi

**Comment by rom1504** [comment]
And update the doc

**Comment by rom1504** [comment]
https://nodejs.org/api/net.html#net_server_listen_port_host_backlog_callback
indeed seems like you right, undefined means ipv6+ipv4 on all addresses

File: src/createServer.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = createServer
 
 function createServer (options = {}) {
   const {
-    host = '0.0.0.0',
+    host,
```
Comment (wvffle) [inline src/createServer.js:16]: This looks good to me. I don't see the point of doing `host = undefined`. It is redundant in my opinion.

**Review by wvffle — APPROVED** [review]

**Comment by rom1504** [comment]
It's not redundant. That marks the parameter as optional.

**Comment by Saiv46** [comment]
Where? There are no `.d.ts` files to mark this parameter as optional.

**Comment by rom1504** [comment]
Just put host = undefined in the function definition

**Comment by rom1504** [comment]
And update the doc to explain the new behavior by default for this parameter

**Comment by rom1504** [comment]
would be good to finish this

**Comment by Saiv46** [comment]
Any progress? `host = undefined` is useless, and `host` parameter just not documented

**Comment by rom1504** [comment]
Yes I'm waiting for you to make the changes.
host = undefined is a clear indication for the reader of the code that is parameter is optional.
host parameter is documented in the API.md file

**Comment by rom1504** [comment]
I don't really understand what's blocking you here. Seems difficult for you to accept code reviews?
I can do the changes myself, but it seems like a good learning opportunity.

**Pull request closed without merging**